tcms/testruns/static/testruns/js/get.js

```diff
@@ -277,12 +277,26 @@ function fileBugFromExecution(execution) {
         const trackerId = $('.one-click-bug-report-form #id-issue-tracker').val()
         jsonRPC('Bug.report', [execution.id, trackerId], result => {
 
+            // close the modal
+            $('#one-click-bug-report-modal button.close').click()
+
+            if (result.rc !== 0) {
+                alert(result.response)
+                return
+            }
+
+            const testExecutionRow = $(`.test-execution-${execution.id}`)
+            animate(testExecutionRow, () => {
+                testExecutionRow.find('.test-execution-hyperlinks').append(renderLink({ is_defect: true, url: result.response }))
+
+                const bugCountEl = testExecutionRow.find('.test-execution-bugs-count')
+                const bugCount = parseInt(bugCountEl[0].innerText)
+                bugCountEl.html(bugCount + 1)
+            })
+
             // unescape b/c Issue #1533
             const targetUrl = result.response.replace(/&amp;/g, '&')
             window.open(targetUrl, '_blank')
-
-            // close the modal
-            $('#one-click-bug-report-modal button.close').click()
```

Comment:
I moved this, so that we close the modal even if the bug creation fails (`result.rc !== 0`) for some reason